confluent_kafka/src/Producer.c

```diff
@@ -402,7 +402,7 @@ static PyObject *Producer_init_transactions (Handle *self, PyObject *args) {
         CallState cs;
         char errstr[256];
         rd_kafka_resp_err_t err;
-        double tmout = -1;
+        double tmout = -1.0f;
```

Comment:
remove the `f` suffix, this will generate a float that is cast to a double.

Reply:
okay, should I do the same for consumer.c or save that for another PR since its unrelated.

Reply:
That's up to you.

Reply:
I'll leave as is for now. I think I want to follow up with a style PR to fix all the inconsistencies